client/components/User-Page/EditProduct.js

```diff
@@ -0,0 +1,150 @@
+import React, { Component } from 'react'
```

Comment:
lot of console.logs here. handleChange functions are solid, but can be re-factored without needing 'name' in input tags